commands/filescommands/commands.go

```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "container",
-			Usage:       "Used for Cloud Files container operations",
+			Usage:       "Storage compartments for your objects/files.",
```

Comment:
compartment feels wrong here. Why not use the name itself?

`Containers for storing objects in the cloud`